### ISSUE TITLE
flow error "unexpected token <" ?

### ISSUE THREAD

**Issue by szhangpitt** [mweststrate/immer#80]
Hi, I have flow 0.63.1 and while checking, I encountered this error

```
Error: node_modules/immer/dist/immer.js.flow:15
 15: declare export default function<S>(
                                    ^ Unexpected token <
```

Do you know what might be the cause? Thanks!

**Comment by marcins** [comment]
Just got this error as well, changing the declarations in `immer.js.flow` to have a function name "fixed" the problem, but I'm not sure of the root cause. That is, I just changed all the exports to:

```js
declare export default function produce<S>( // ...
```

**Comment by mweststrate** [comment]
Fix has been released in 1.0.0

Op di 30 jan. 2018 om 00:14 schreef Marcin Szczepanski <
notifications@github.com>:
